src/plugins/formulas/engine/settings.js

```diff
@@ -0,0 +1,81 @@
+import { PLUGIN_KEY } from '../formulas';
+
+// TODO: refine the list of default HF settings
+const DEFAULT_SETTINGS = {
+  licenseKey: 'internal-use-in-handsontable',
+
+  // default engine configuration
+  binarySearchThreshold: 20,
+  matrixDetection: true,
+  matrixDetectionThreshold: 100,
+  useColumnIndex: false,
+  useStats: false,
+
+  // desired UX
+  evaluateNullToZero: true, // excel compatibility
+  precisionEpsilon: 1e-13, // excel compatilibity
+  precisionRounding: 14, // excel compatilibity
+  smartRounding: true, // excel compatilibity
+  leapYear1900: true, // excel and lotus123 leap year bug compatiblity
+  nullDate: {
+    year: 1899,
+    month: 12,
+    day: 31
+  }, // same as above, not sure if correct
+  nullYear: 30,
```

Comment:
Should be ok: https://github.com/handsontable/hyperformula/issues/58#issuecomment-581930373
```suggestion
  leapYear1900: true,
  nullDate: {
    year: 1899,
    month: 12,
    day: 31
  },
  nullYear: 30,
```